move2kube/README.md

```diff
@@ -0,0 +1,55 @@
+Move2kube
+===========
+
+# Configuration
```

Comment:
A bit broader question than this change. How should we separate workflows from the orchestrator?
Let's assume I would like to install my workflow to a specific namespace. I am making assumption that all workflow dependencies like move2kube operator, my custom functions or any other secrets I need would need to be created/applied in this specific namespace. Am I correct?

cc @masayag

Reply:
First, we should have the Cluster-scope services feature included in the operator to support workflows from various namespaces to work against the same DI and job-service.

Next, at least for now, we should split the workflow deployment from the orchestrator deployment since if a workflow is started before DI, the event to publish the workflow can not hit the target and therefore the DI will not be aware of that workflow.

IIUC, the sonataflow-operator watches all sonataflows in all namespaces.
The workflows communicate with the DI and JobService via http/cloud-events (in the future kafka).
However, mounting secrets and configmaps to the workflow deploy requires them (k8s restriction) to exist in the same namespace.